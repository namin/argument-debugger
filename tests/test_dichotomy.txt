Either we cut social programs or the economy will collapse.
We cannot let the economy collapse.
Therefore, we must cut social programs.

Either the defendant is guilty or innocent.
The evidence shows they are not innocent.
Therefore, they must be guilty.

Either God exists or God does not exist.
There is no proof God exists.
Therefore, God does not exist.