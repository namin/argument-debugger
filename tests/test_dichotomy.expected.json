{
  "description": "Tests false dichotomy detection - first should flag, others shouldn't",
  "must_find": [],
  "may_find": ["false_dichotomy", "unsupported_premise", "missing_link"],
  "must_not_find": ["circular"]
}